purview/base.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import json
+
+
+class BaseAPI(object):
+    """
+    Base API class to standardize the main API functionality
+    """
+
+    def serialize(self, node_obj):
```

Comment:
This should be a function on the base class our StructuredNode classes inherit from.

Reply:
I didn't quite get this one. Sorry!

Reply:
Yes, create a base class that inherits from `StructuredNode` and then have our current models inherit from that base class. Then create the `serialize` function on that new base class. This will allow us to just call `serialize()` on any Neo4j node.